getting-started/photometry.rst

```diff
@@ -0,0 +1,176 @@
+We'll accomplish this with :command:`measureCoaddSources.py`:
+
+.. code-block:: bash
+
+   measureCoaddSources.py DATA --rerun coaddPhot --id filter=HSC-R
+
+And repeat with the ``HSC-I``-band coadd:
+
+.. code-block:: bash
+
+   measureCoaddSources.py DATA --rerun coaddPhot --id filter=HSC-I
+
+.. .. tip::
+.. 
+..    The :command:`measureCoaddSources` command line task produces ``deepCoadd_meas`` datasets in the Butler data repository.
+..    Because the same merged detection catalog is used for every filter, the ``HSC-R`` and ``HSC-I``-band ``deepCoadd_meas`` tables have consistent rows.
+..    We'll see how to access these tables later.
+
+.. Data product is deepCoadd_meas
+
+.. _getting-started-tutorial-merge-coadds:
+
+Merging multi-band source catalogs from coadds
+----------------------------------------------
+
+The previous step created measurement catalogs for each patch in both the ``HSC-R`` and ``HSC-I`` bands.
+We get even more complete and consistent multi-band photometry by measuring the same source in multiple bands at a fixed position (the forced photometry method) rather than fitting the source's location individually for each band.
+
+For forced photometry we want to use the best position measurements for each source, which could be from different filters depending on the source.
+We call the filter that measures a source best the **reference filter**.
+Let's use the :command:`mergeCoaddMeasurements.py` command to create a table that identifies the reference filter for each source in the tables we created with the previous step.
+Run:
+
+.. code-block:: bash
+
+   mergeCoaddMeasurements.py DATA --rerun coaddPhot --id filter=HSC-R^HSC-I
+
+.. Data product is deepCoadd_ref
+
+.. _getting-started-tutorial-forced-coadds:
+
+Running Forced photometry on coadds
+-----------------------------------
+
+Now we have accurate positions for sources in the patches.
+Let's re-measure the coadds using these fixed source positions (the forced photometry method) to create the best possible photometry of sources in our coadds.
+Run:
+
+.. code-block:: bash
+
+   forcedPhotCoadd.py DATA --rerun coaddPhot:coaddForcedPhot --id filter=HSC-R
+
+Also run forced photometry on the ``HSC-I``-band coadds:
+
+.. code-block:: bash
+
+   forcedPhotCoadd.py DATA --rerun coaddForcedPhot --id filter=HSC-I
+
+.. Data product is deepCoadd_forced_src
+
+The :command:`forcedPhotCoadd.py` command creates table datasets called ``deepCoadd_forced_src`` in the Butler repository.
+In the next part of this tutorial we'll see how to work with these tables.
+
+.. note::
+
+   You can also try the :command:`forcedPhotCcd.py` command to apply forced photometry to individual exposures, which may in principle yield better measurements.
+   :command:`forcedPhotCcd.py` doesn't currently deblend sources, though.
+   Thus forced coadd photometry, as we performed here, provides the best photometry of coadded sources.
```

Comment:
That last sentence is true independent of the previous ones (which are visit-level measurements).